test/run.d

```diff
@@ -145,7 +181,29 @@ auto predefinedTargets(string[] targets)
         return testPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
     }
 
-    Appender!(string[]) newTargets;
+    static Target createUnitTestTarget()
+    {
+        Target target = {
+            args: [getEnvironment["DMD"], "-run", "unit_test_runner.d"]
```

Comment:
Why not use `hostDMD` here like we do to build the other tools? That way you don't need to rebuild the binary every time when the DMD binary changes.

Reply:
> Why not use `hostDMD` here like we do to build the other tools?

I can try `hostDMD`.

> That way you don't need to rebuild the binary every time when the DMD binary changes.

Why would the binary need to be rebuilt when the DMD binary changes?